spec/stat_tracker_spec.rb

```diff
@@ -17,57 +17,57 @@
   end
   
   describe 'initialize' do
-    it 'exists' do
+    xit 'exists' do
```

Comment:
reminder to un 'x' these test before final pr

Reply:
Good call!